subprojects/build-init/src/main/java/org/gradle/buildinit/plugins/internal/JavaApplicationProjectInitDescriptor.java

```diff
@@ -28,8 +29,12 @@ public JavaApplicationProjectInitDescriptor(TemplateOperationFactory templateOpe
     protected void configureBuildScript(BuildScriptBuilder buildScriptBuilder) {
         super.configureBuildScript(buildScriptBuilder);
         buildScriptBuilder
-            .plugin("Apply the application plugin to add support for building an application", "application")
-            .configuration("Define the main class for the application", "mainClassName = 'App'");
+            .plugin(
```

Comment:
:x: This duplication (see GroovyApplicationProjectInitDescriptor) doesn't seem necessary, is it?

Reply:
Factored as a utility method on `BuildScriptBuilder` ✅